### PR TITLE
spike: normalization

### PR DESCRIPTION
**Pull request by 121watts**
WIP: Link to [RFC](https://github.com/influxdata/idpe/issues/5591)

### CODE REVIEW

File: ui/src/dashboards/actions/n.dashboards.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// Libraries
+import {normalize, NormalizedSchema} from 'normalizr'
```
Comment (Asalem1) [inline ui/src/dashboards/actions/n.dashboards.ts]: can we drop the `e` from normalize to keep up to "cool" standards

File: ui/src/dashboards/components/dashboard_index/Table.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ class DashboardsTable extends PureComponent<Props, State> {
           emptyState={this.emptyState}
           className={dashboards.length ? 'dashboards-card-grid' : ''}
         >
-          {dashboards.length ? (
+          {!!dashboards.length && (
```
Comment (Asalem1) [inline ui/src/dashboards/components/dashboard_index/Table.tsx]: nice

File: ui/src/dashboards/reducers/n.dashboards.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  state: DashboardsState['byID'] = {},
+  action: DashboardActionTypes
+): DashboardsState['byID'] => {
+  switch (action.type) {
+    case SET_DASHBOARDS: {
+      if (!action.normalized) {
+        return state
+      }
+
+      return {...state, ...action.normalized.entities.dashboards}
```
Comment (Asalem1) [inline ui/src/dashboards/reducers/n.dashboards.ts]: can we safeguard against some of these assumptions? Maybe using `get` from lodash?

File: ui/src/store/configureStore.ts [hunk 4]
```diff
@@ -49,37 +55,42 @@ type ReducerState = Pick<AppState, Exclude<keyof AppState, 'timeRange'>>
...
   userSettings: userSettingsReducer,
-  members: membersReducer,
-  cloud: combineReducers<{limits: LimitsState}>({limits: limitsReducer}),
-  checks: checksReducer,
-  rules: rulesReducer,
-  endpoints: endpointsReducer,
+  variables: variablesReducer,
+  variableEditor: variableEditorReducer,
   VERSION: () => '',
+  views: viewsReducer,
```
Comment (Asalem1) [inline ui/src/store/configureStore.ts]: Thanks for doing this

File: ui/src/dashboards/components/dashboard_index/Table.tsx (hunk 2 above)
Comment (TCL735) [reply ui/src/dashboards/components/dashboard_index/Table.tsx]: https://github.com/influxdata/clockface/issues/421

**Comment by russorat** [comment]
@121watts should this be closed?

**Comment by 121watts** [comment]
yes

Andrew

**Pull request closed without merging**